pyproject.toml

```diff
@@ -150,3 +135,22 @@ skip_covered = true
 [build-system]
 requires = ["poetry-core>=1.0.0", "setuptools"]
 build-backend = "poetry.core.masonry.api"
+
+[project]
+name = "geoh5py"
+description = "Python API for geoh5, an open file format for geoscientific data"
+license = "LGPL-3.0-or-later"
+keywords = ["geology", "geophysics", "data", "interoperability"]
+version = "0.12.0-alpha.1"
+dynamic = ["classifiers", "requires-python"]
```

Comment:
👍 dynamic classifiers

I was not aware of the following
(from https://python-poetry.org/docs/pyproject/#classifiers)
> Note that suitable classifiers based on your python requirement and license are not automatically added for you if you define classifiers statically in the project section.
> 
> If you want to enrich classifiers automatically, you should add classifiers to dynamic and use the tool.poetry section instead.